blocks/init/components/button/components/button-options.js

```diff
@@ -0,0 +1,97 @@
028932l-2.06146238-2.06161496c-.30899047-.30899051-.30899047-.81176758 0-1.12075809l1.55441285-1.55456543c.14968871-.14968871.3488159-.23208617.56045531-.23208617.06561281 0 .12954711.00885008.19149781.02410891l3.59375004-3.59359742c.3089904-.30899047.8117675-.30899047 1.1209106 0l.6977844.69793699h3.2247925c.1618957 0 .2929688.13107301.2929688.29296875 0 .16174316-.1310731.29296875-.2929688.29296875h-2.638855l3.8491822 3.84902957c.3089904.30899047.3089904.81192015 0 1.12091062l-3.5940552 3.5940552c.0643921.2607727-.0041199.5480957-.2076722.7514954l-1.4912414 1.491394v1.4624023c0 .6033326.4908752 1.0940552 1.0942078 1.0940552.6031799 0 1.0940551-.4907226 1.0940551-1.0940552v-1.9816589c0-.9263611.7536316-1.6799927 1.6799927-1.6799927s1.6801453.7536316 1.6801453 1.6799927v6.516571c0 .6033326.4907226 1.0942078 1.0940552 1.0942078.6033325 0 1.0940552-.4908752 1.0940552-1.0942078v-3.9518738c0-.1618957.1312255-.2929687.2929687-.2929687zm-8.5760498-13.42895509-3.51318364 3.51318359.42099.42099 2.33657836-2.33642578c.11428838-.1144409.29983528-.1144409.41427618 0 .1144409.11444094.1144409.29983524 0 .41442871l-2.33642583 2.33642578.31570434.31570434.59371949-.59356687c.11428832-.11444094.2998352-.11428833.41427613 0 .11444094.11444093.11444094.29998781 0 .41427613l-.59371949.59356687 2.07489016 2.07489016.5937195-.59356691c.1142883-.11428833.2998352-.11428833.4142761 0 .1144409.11444093.1144409.29983523 0 .41442871l-.5935669.59356691 1.3703918 1.37039184 3.5131836-3.5131836c.0805664-.08041383.0805664-.21163941 0-.29205324l-5.132904-5.13305664c-.0805664-.08056641-.2116394-.08056641-.2922058 0zm-.2230835 11.32522589c.0390625.0390625.0907898.0605774.1460266.0605774s.1071167-.0215149.1461792-.0605774l1.5544128-1.5544129c.0805664-.0805664.0805664-.2116394 0-.2922058l-5.67687983-5.67687989c-.0390625-.0390625-.09094242-.06057738-.14617922-.06057738-.05508422 0-.1069641.02151488-.1460266.06057738l-1.55456543 1.55441286c-.08056641.0805664-.08056641.21163941 0 .29220582zm-3.63372803-5.14236456 3.09921262 3.09921265c.11444091.11444091.11444091.29983521 0 .41442871-.05722043.0570679-.13214109.0857544-.20706176.0857544-.07507324 0-.1499939-.0286865-.20721437-.0857544l-3.09936524-3.09936523c-.1144409-.1144409-.1144409-.29983519 0-.41427613.11444094-.1144409.29998781-.1144409.41442875 0zm10.77133173.29129031c.1618958 0 .2929688.13107297.2929688.29296875 0 .16189574-.131073.29296875-.2929688.29296875-.1618957 0-.2944946-.13107301-.2944946-.29296875 0-.16189578.1295471-.29296875.2914429-.29296875zm.2413941.80078125h.0030518.0036621c.1602172 0 .290985.12878418.2929687.28930664.0019836.16174316-.1275635.29449461-.2893066.29663086h-.0038147c-.1600647 0-.2923585-.12878418-.2943421-.28930664-.0019836-.16174316.1258851-.29449465.2877808-.29663086zm-1.8183899.90255734h.0030517c.1618958 0 .2929688.13107301.2929688.29296875 0 .16174321-.131073.29296875-.2929688.29296875-.1618957 0-.2944946-.13122554-.2944946-.29296875 0-.16189574.1295471-.29296875.2914429-.29296875z" /></svg>} />
```

Comment:
This svg cannot be imported using something like this: https://www.npmjs.com/package/@svgr/webpack? Or is this something WP specific?

Reply:
I will have to test this but I will do it in the next PR this is a huge one. tnx